### ISSUE TITLE
msg.percentage not available

### ISSUE THREAD

**Issue by phaeton82** [datech/node-red-contrib-amazon-echo#49]
SUMMARY
Update with percentage does not appear in repo node-red. It's not a part of the MSG object. Could you update the version in Node-Red to support the percentage added in: Output original percentage request of brightness value #45 

-- 

update: installed the version direcly from github.

**Comment by phaeton82** [comment]
After figuring out how to install this flow directly from Github I have the percentage. If I  give alexa the command for percentage the msg.percentage gives a different % below 22. It deducts one percentage. I suspect there are some rounding issues. For lights this is not a big issue, for setting the thermostat through node-red it is.

**Comment by 3ative** [comment]
@phaeton82 
I use this (in a Function Node) to convert to %
```yaml
temp = Math.round( (msg.bri/254)*100);
msg.payload = temp;
return msg 
```

**Comment by datech** [comment]
@phaeton82 I've tested it on my Echo Dot 3, and it works as expected. What kind of Echo do you use?

**Comment by phaeton82** [comment]
I will reinstall and try again. later this week.

**Comment by datech** [comment]
@phaeton82 It was just released in v0.1.9
